src/view/uielement.js

```diff
@@ -82,9 +83,51 @@ export default class UIElement extends Element {
 	}
 }
 
+/**
+ * Assign key observer which will move cursor over a ui element if right arrow key is pressed and selection is before
```

Comment:
Besides, this function's description should start with some general introduction. Then it can start describing particular quirks.